src/XenCert/StorageHandlerUtil.py

```diff
@@ -935,7 +935,7 @@ def verify_data_on_vdi(session, vbd_ref, start_sec, end_sec):
 
         xencert_print('about to read from device: %s' % device)
 
-        expect = BUF_PATTERN.encode()
+        expect = BUF_PATTERN.encode('utf-8')
```

Comment:
BUF_PATTERN is constant, so you can encode it when it is defined.
```
BUF_PATTERN = (CHAR_SEQ + CHAR_SEQ).encode('utf-8')
```

Reply:
Has encode the pattern